testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-delta-lake-databricks-http-metastore/hive.properties

```diff
@@ -0,0 +1,13 @@
+connector.name=hive
+hive.metastore=glue
+hive.metastore.glue.region=${ENV:AWS_REGION}
+# We need to give access to bucket owner (the AWS account integrated with Databricks), otherwise files won't be readable from Databricks
+hive.s3.upload-acl-type=BUCKET_OWNER_FULL_CONTROL
+hive.allow-drop-table=true
+hive.non-managed-table-writes-enabled=true
+# Required by some product tests
```

Comment:
Is this actually the case or is this copy-paste?

Reply:
This is a copy-paste. I was not sure if we must configure a hive.properties for singlenode tests, so I just reused it from another test. I will remove that comment.